upstream/upaws/provision_data.go

```diff
@@ -180,29 +181,35 @@ func generateFunctionNames(manifest *apps.Manifest, functions []FunctionData) ma
 }
 
 func (pd *ProvisionData) Validate() error {
+	var result error
 	if pd.Manifest == nil {
-		return errors.New("no manifest")
+		result = multierror.Append(result,
+			errors.New("no manifest"))
 	}
 	if err := pd.Manifest.Validate(); err != nil {
```

Comment:
If `pd.Manifes` is `nil`, calling `Validate()` on it will panic. Should the first if `return`?

Reply:
Thanks for catching, I went a little too "replace-y"

Reply:
Fixed it in the feature branch `lev-faasd`